backend/crud.py

```
@@ -0,0 +1,32 @@
+from typing import List, Sequence
+
+from sqlmodel import Session, select
+
+from backend.models import Hero
+from backend.schemas import HeroCreate, HeroRead
```

Ruff (0.11.9)

`backend.schemas.HeroRead` imported but unused

Remove unused import: `backend.schemas.HeroRead`

(F401)